### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Ishika11**

### CODE REVIEW

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images2.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      <h1>Ishika's Image Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images2/Genn.jpg"><img src="images2/Genn.jpg"/></a>
+        <a href="images2/Sheryl.jpg"><img src="images2/Sheryl.jpg"/></a>
+        <a href="images2/Sun.jpg"><img src="images2/sun.jpg"/></a>
+        <a href="images2/book.jpg"><img src="images2/book.jpg"/></a>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images2.html]: nit: Add a newline. Always a good idea to have an empty newline at end of file.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images2.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images2.html]: Just edit the same file. Keep the same name i.e. images.html

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/random.html [hunk 3]
```diff
@@ -2,16 +2,15 @@
...
   </head>
   <body>
     <div id="content">
-      <h1>Random Stanley 🐈</h1>
-      <p>Click here to see a random picture of Stanley:</p>
-      <button onclick="randomizeImage()">Meow</button>
-      <div id="random-image-container"></div>
+      <h1>My pic!</h1>
+      <p>Nice to meet you :)</p>
+      <img src="images2/mypic.jpg"/>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/random.html]: When you change filename to images, don't forget to update path here.

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/random.html [hunk 4]
```diff
@@ -2,16 +2,15 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Random Stanley 🐈</title>
+
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/random.html]: Add title back. You can change text in title but should have a title.

**Review by abhatia2107 — CHANGES_REQUESTED** [review]

**Comment by abhatia2107** [comment]
Would also request you to make some css changes as part of this PR. Try playing around different styles.

**Comment by Ishika11** [comment]
> Add title back. You can change text in title but should have a title.

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images.html [hunk 5]
```diff
@@ -2,27 +2,19 @@
...
+        <a href="images/Genn.jpg"><img src="images/Genn.jpg"/></a>
+        <a href="images/Sheryl.jpg"><img src="images/Sheryl.jpg"/></a>
+        <a href="images/Sun.jpg"><img src="images/sun.jpg"/></a>
+        <a href="images/book.jpg"><img src="images/book.jpg"/></a>
       </div>
     </div>
   </body>
-</html>
+  <br>
+</html>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images.html]: newline

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    <div id="content">
+      <h1 style="color: Orange;">Helix - Ishika Kumar</h1>
+      <div class="image-thumbnails">
+        <a href="images/Helix.jpg"><img src="images/Helix.jpg"/></a>
+        
+      </div>
+    </div>
+  </body>
+  <br>
+</html>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html]: newline

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My_book</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style_my_book.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1 style="color: Orange;">Helix - Ishika Kumar</h1>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html]: Avoid inline style

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+      <div class="image-thumbnails">
+        <a href="images/Helix.jpg"><img src="images/Helix.jpg"/></a>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html:13]: add alt text

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_index.css [hunk 9]
```diff
@@ -0,0 +1,23 @@
+p
+{
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 30px;
+  color: black;
+}
+h1
+{
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_index.css:8]: be consistent with placement of {
Either put it in newline or at end of last line. Fine with anything but would appreciate consistency.

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_random.css [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+p{
+    color: tomato;
+    font-size: 20px;
+    font-family: 'Times New Roman', Times, serif;
+}
+h1{
+    color: slateblue;
+    font-size: 30px;
+    font-family: 'Times New Roman', Times, serif;
+}
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_random.css]: newline

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_my_book.css [hunk 11]
```diff
@@ -0,0 +1,8 @@
+body {
+    background-color: white;
+}
+h1{
+    color: orange;
+    font-size: 40px;
+    font-family: 'Times New Roman', Times, serif;
+}
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_my_book.css]: newline

**Review by abhatia2107 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**